tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/grammar/BallerinaParser.g4

```diff
@@ -163,7 +163,7 @@ finiteTypeUnit
 
 typeName
     :   simpleTypeName                                                                          # simpleTypeNameLabel
-    |   typeName (LEFT_BRACKET (integerLiteral | sealedLiteral)? RIGHT_BRACKET)+                # arrayTypeNameLabel
+    |   typeName (LEFT_BRACKET (integerLiteral | MUL)? RIGHT_BRACKET)+                          # arrayTypeNameLabel
```

Comment:
This file is kept as a reference to identify the changes done to the grammar. These changes are later applied to the IDEA plugin. So better to revert this change.

Reply:
+1, @NatashaWso2 @NipunaRanasinghe Please note.

Reply:
Reverted the change with https://github.com/ballerina-platform/ballerina-lang/pull/14163/commits/ce24b27c4f07bb71237b044370fd65494aab7f74